### DISCUSSION TITLE
WPF - Run browser minimized?

### DISCUSSION THREAD

**Discussion by kevinvenclovas** [cefsharp/CefSharp#4292] [General]
Is there an option that cefbrowser keep running when the application minimized? Im using cef in my .net core WPF application and need to access browser screen also wehen my window is minimized

**Comment by amaitland** [comment]
In future please use https://github.com/cefsharp/CefSharp/discussions to ask questions

In WPF you can subclass `ChromiumWebBrowser` and override `OnBrowserWasHidden`. Don't call the base method and the browser will stay active.

https://github.com/cefsharp/CefSharp/blob/cefsharp/107/CefSharp.Wpf/ChromiumWebBrowser.cs#L1762

**Reply by kevinvenclovas** [reply]
Thank you!
